generators/build/templates/Rakefile

```diff
@@ -24,7 +18,22 @@ namespace :static do
   end
 end
 
-task :prereqs => [ 'presteps:clean_up' ]
+namespace :integration do
+  task :ensure do
+    success = system ("dep ensure")
+    if not success 
+      raise "ERROR: Dep ensure failed!\n".red
+    end
+  end
+  task :test do
+    success = system ("go test -v ./test/ -timeout 20m -args azureuser ~/.ssh/id_rsa")
+    if not success 
+      raise "ERROR: Go test failed!\n".red
+    end
+  end
+end
```

Comment:
Why do you pass the SSH key here?

Reply:
Oh I used code from compute module. Will remove it.